.github/workflows/stockfish.yml

```
@@ -139,49 +139,53 @@ jobs:
   # Testing Jobs
   IWYU:
     name: IWYU
+    if: false
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)